src/interfaces/restoration-interfaces/restoration-interface.ts

```diff
@@ -7,16 +7,17 @@ import { ContactPointIF } from '@bcrs-shared-components/interfaces/'
 // Ref: https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/restoration.json
 //
 export interface RestorationIF {
```

Comment:
This interface still doesn't match the schema (required vs non-required properties).

Reply:
Required properties are :
"type",
 "approvalType",
 "offices",
"parties",
 "contactPoint"

Reply:
Looks good now. Thanks.